### PR TITLE
take "+ tab" out of the monospaced codeword

### PR DESCRIPTION
**Pull request by sonmaximum**
### Proposed changes

Correct what seems to be a typo in the completion.md readme where "+ tab" was included in the monospaced codeword instruction.

### CODE REVIEW

**Review by usha-mandya — DISMISSED** [review]
Thank you :)

File: compose/completion.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Make sure you have [installed `oh-my-zsh`](https://ohmyz.sh/) on your computer.
 Add `docker` and `docker-compose` to the plugins list in `~/.zshrc` to run
 autocompletion within the oh-my-zsh shell. In the following example, `...`
 represent other Zsh plugins you may have installed. After that, type `source ~/.zshrc` to bring the changes. 
-To test whether it is successful, type `docker ps + tab`.
+To test whether it is successful, type `docker ps` + tab.
```
Comment (thaJeztah) [inline compose/completion.md:81]: Wondering if the "+ tab" is confusing; I think it's common to use something like;

```suggestion
To test whether it is successful, type `docker ps <TAB>`.
```

Comment (usha-mandya) [reply compose/completion.md:81]: To test whether it is successful, type `docker ps` and then press the Tab key.

Comment (thaJeztah) [reply compose/completion.md:81]: Yup, works for me (or `TAB` (formatted as code))

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**